src/utils/prompt-for-missing-options.ts

```diff
@@ -69,6 +70,7 @@ export async function promptForMissingOptions(options: RawOptions): Promise<Opti
     dev: options.dev ?? defaultOptions.dev,
     extensions: [options.solidityFramework ?? answers.solidityFramework].filter(ext => Boolean(ext) && ext !== "none"),
     externalExtension: options.externalExtension,
+    help: options.help,
```

Comment:
Umm feels wired to have this and returned by `promptForMissingOptions` function and pass it further. 

Because actually this will never happen right ? Because of : 


```ts
    if (rawOptions.help) {
      showHelpMessage();
      return;
    }
     const options = await promptForMissingOptions(rawOptions);

```

Like I see the help should be only present in `rawOptions` (RawOptions type) and is not required / won't be when we parse and construct final options(work done by `promptForMissingOptions` func. 

So I would update the types as: 

```diff
// type.ts

type BaseOptions = {
  project: string | null;
  install: boolean | null;
  dev: boolean;
  externalExtension: ExternalExtension | null;
-  help: boolean;
};

export type RawOptions = BaseOptions & {
  solidityFramework: SolidityFramework | "none" | null;
+  help: boolean;
};
```

And remove `help: options.help` from this file

Reply:
Thanks, you're right! When I wrote it at the start I thought I need to use it later and forgot to remove later [f885075](https://github.com/scaffold-eth/create-eth/pull/63/commits/f885075add585f1b9013cc166cafd296469099fb)